doc/05_Objects/03_Working_with_PHP_API.md

```diff
@@ -446,15 +446,18 @@ on `\Doctrine\DBAL\Query\QueryBuilder`.
 ```php
 <?php
 // get all news with ratings that is stored in a not Pimcore related table
- 
-/** @var \Pimcore\Model\DataObject\Listing\Dao|\Pimcore\Model\DataObject\News\Listing $list */
+
 $list = new Pimcore\Model\DataObject\News\Listing();
```

Comment:
```suggestion
$list = new \Pimcore\Model\DataObject\News\Listing();
```

Reply:
Because the example has no namespace this should be valid without leading `\`.  But I think we can set use statements for `Listing` and `QueryBuilder`.